.vale/styles/RedHat/PascalCamelCase.yml

```diff
@@ -0,0 +1,42 @@
+---
+extends: existence
+scope: sentence
+ignorecase: false
+nonword: true
+level: warning
```

Comment:
Do we want to change this to `suggestion` temporarily while we put this through a testing phase?